### PR TITLE
chore: 🔧 update maintainers

### PR DESCRIPTION
**Pull request by mloiseleur**
### What does this PR do?

Update maintainers based on top 3 contributors displayed with [monocle](https://github.com/change-metrics/monocle) using this config file

```yaml
workspaces:
  - name: monocle
    crawlers:
      - name: github-traefik
        provider:
          github_organization: traefik
          github_repositories:
            - traefik-helm-chart
        update_since: '2022-01-01'
```

**Peers Strengh**

peers-strenght

**Changes created**

changes-created

**Review authors**

review-authors

### Motivation

Up-to-date metadata.

### CODE REVIEW

**Review by darkweaver87 — APPROVED** [review]
LGTM
Per our previous discussions, please remove my rights on that repo.

**Comment by mloiseleur** [comment]
The stats provided by GitHub are not showing review contributions.
I found a way to get those stats using [monocle](https://github.com/change-metrics/monocle). 
=> Wdyt ?

**Review by darkweaver87 — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,4 +27,41 @@ While encouraging contributions, the philosophy leads to avoid introducing:
...
 * dashboard exposition tuning
-* helm chart variables that shortcuts/expose static or dynamic Traefik configuration
\ No newline at end of file
+* helm chart variables that shortcuts/expose static or dynamic Traefik configuration
+
+# Statistics
+
+Once a year, [monocle](https://github.com/change-metrics/monocle) is used to gather statistics on this project.
+
+They were gathered with this config file:
```
Comment (svx) [inline CONTRIBUTING.md:36]:
```suggestion
They are gathered with this config file:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,4 +27,41 @@ While encouraging contributions, the philosophy leads to avoid introducing:
 * specific use cases
 * third party CRD
 * dashboard exposition tuning
-* helm chart variables that shortcuts/expose static or dynamic Traefik configuration
\ No newline at end of file
+* helm chart variables that shortcuts/expose static or dynamic Traefik configuration
```
Comment (svx) [inline CONTRIBUTING.md:30]:
```suggestion
- Helm chart variables that shortcuts/expose static or dynamic Traefik configuration
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -27,4 +27,41 @@ While encouraging contributions, the philosophy leads to avoid introducing:
 * specific use cases
 * third party CRD
 * dashboard exposition tuning
```
Comment (svx) [inline CONTRIBUTING.md:29]:
```suggestion
- Dashboard exposition tuning
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -27,4 +27,41 @@ While encouraging contributions, the philosophy leads to avoid introducing:
 * specific use cases
 * third party CRD
```
Comment (svx) [inline CONTRIBUTING.md:28]:
```suggestion
- Third party CRD
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -27,4 +27,41 @@ While encouraging contributions, the philosophy leads to avoid introducing:
 * specific use cases
```
Comment (svx) [inline CONTRIBUTING.md:27]:
```suggestion
- Specific use cases
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -13,7 +13,7 @@ Please read the [testing guidelines](./TESTING.md) to learn how testing is done
 
 ## Guidelines
 
-According to the Traefik HelmChart [philosophy](./README.md#philosophy), 
+According to the Traefik HelmChart [philosophy](./README.md#philosophy),
 the guidelines for future evolutions are:
 
 * fix bugs
```
Comment (svx) [inline CONTRIBUTING.md:19]:
```suggestion
- Fix bugs
```

**Comment by ldez** [comment]
why do need this kind of update?

**Comment by mloiseleur** [comment]
> why do need this kind of update?

The goal is to provide up-to-date metadata. Last update of this field was 4 years ago. 

To be honest, I'm unsure how to proceed. This kind of PR is weird. 
Even though previous maintainers are not active anymore, they have done a great job when they were. 
=> I have no idea on how to represent that in this Chart.

I'm more than open to suggestion on this PR :pray: .

**Review by darkweaver87 — APPROVED** [review]
LGTM

**Comment by ldez** [comment]
Being a maintainer is not related to activity, this is not a popularity or activity contest.

This PR expresses something that is not the rule in the organization to define maintainers.

The maintainers are maintainers and should express explicitly the decision to stop being a maintainer.

There is no need for this PR.

If you want to be up-to-date, the best way is to add all the current maintainers.
This PR doesn't provide an exhaustive list of the maintainers of the hemlchart.

**Comment by mloiseleur** [comment]
> Being a maintainer is not related to activity, this is not a popularity or activity contest.

I don't agree with you on this. [Activity is the key](https://github.com/traefik/traefik/blob/master/docs/content/contributing/maintainers-guidelines.md#maintainers-activity) to both become or not a maintainer. 

I'll probably also be removed by a new maintainer in the future. That's life.

**Comment by ldez** [comment]
> I don't agree with you on this. [Activity is the key](https://github.com/traefik/traefik/blob/master/docs/content/contributing/maintainers-guidelines.md#maintainers-activity) to both become or not a maintainer.

I don't agree with your reading of this section.

**Pull request merged**